_sass/layout/_search.scss

```diff
@@ -0,0 +1,23 @@
+.search__wrapper {
+  margin-top: 20px;
+  margin-bottom: 50px;
+}
+
+.searchbar {
+  margin-bottom: 10px;
+
+  input {
+    height: 20px;
+  }
+}
+.ais-Pagination {
+  margin-top: 20px;
+}
+
+.ais-Hits-item {
+  width: 100% !important;
+}
+
+.ais-RefinementList-labelText {
+  margin-left: 5px;
```

Comment:
Shouldn't this be 5 * $spacing-base?

Reply:
I think this would actually be more like `0.5 * $spacing-base`, since it's very small. This seemed specific enough that I wasn't sure it was needed, but you can change it if you'd like. I believe you can also do `0.5rem` instead, but I didn't test that. Feel free to play around with it and see what looks best to you!

Reply:
oops, it is 5 rather than 50.

Reply:
I'm not sure where this gets used, I'll leave it for @MrVantastic

Reply:
This is to add some spacing in between the checkboxes and the text for the refinement options: 
<img width="201" alt="Screen Shot 2020-07-28 at 4 30 56 PM" src="https://user-images.githubusercontent.com/13214848/88739595-c0491e80-d0ef-11ea-999e-ca9f5c82b1eb.png">

But I agree, we should use variables instead of actual numbers whenever possible. I'll make a change.

Reply:
@mikeubell This class comes from Algolia's css (the stylesheet that is imported in `search.html`) and is applied to one of the components that we're using from Algolia - we aren't applying this class directly to a component that we've built. @MrVantastic added a small margin to Algolia's class to improve how it looks in our use case here.